src/components/SecondaryPanes/ChromeScopes.js

```diff
@@ -212,7 +208,9 @@ class Scopes extends Component<Props> {
 export default connect(
   state => ({
     isPaused: getIsPaused(state),
-    loadedObjects: getLoadedObjects(state),
+    loadedObjects: () => {
+      throw new Error("This is not implemented.");
```

Comment:
I believe this file is not run, and the `getLoadedObjects` function did not exist. I'm not sure of the state of this, so I threw an error if someone actually tries to use it.